pkg/controllers/nsxserviceaccount/nsxserviceaccount_controller.go

```diff
@@ -201,7 +201,8 @@ func (r *NSXServiceAccountReconciler) validateRealized(count uint16, ca []byte,
 	// Validate ca at first time
 	// Validate client cert every GCValidationInterval
 	if count == 0 {
-		for _, nsxServiceAccount := range nsxServiceAccountList.Items {
+		for _, account := range nsxServiceAccountList.Items {
+			nsxServiceAccount := account
 			if nsxserviceaccount.IsNSXServiceAccountRealized(&nsxServiceAccount.Status) {
```

Comment:
not sure what's the update fix? it seems they are the same.

Reply:
"G601: Implicit memory aliasing in for loop. (gosec)" error detected by golangci-lint tool for the original way.